datagrepper.spec

```diff
@@ -0,0 +1,278 @@
+# https://bugzilla.redhat.com/show_bug.cgi?id=955781
+%global pypi_name datagrepper
+%global pypi_version 1.0.0
+
+Name:           %{pypi_name}
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A webapp to query fedmsg history
+
+License:        GPLv2+
+URL:            https://github.com/fedora-infra/datagrepper
+Source0:        %{pypi_source %{pypi_name}}
+
+BuildArch:      noarch
+
+BuildRequires:  make
+BuildRequires:  python3-arrow
+BuildRequires:  python3-datanommer-models
+BuildRequires:  python3-devel
+BuildRequires:  python-docutils
+BuildRequires:  python3-fedmsg-core
+BuildRequires:  python3-fedmsg-meta-fedora-infrastructure
+BuildRequires:  python3-flask
+BuildRequires:  python3-flask-sqlalchemy
+BuildRequires:  python3-freezegun
+BuildRequires:  python3-mock
+BuildRequires:  python3-nose
+BuildRequires:  python3-pygal
+BuildRequires:  python3-pytest-cov
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sqlalchemy
+
+%description
+A webapp to retrieve historical information about messages on the fedmsg
+bus.  It is a JSON api for the datanommer message store.
+
+%prep
+%autosetup -n %{pypi_name}-%{pypi_version}
+# makes build_sphinx find it
+# cannot move, we need to keep sources there, as they are used on runtime
+cp -a %{pypi_name}/docs .
+
+%generate_buildrequires
+%pyproject_buildrequires
+
+%build
+%pyproject_wheel
+make %{?_smp_mflags} -C %{pypi_name}/docs
+
+%install
+%pyproject_install
+%pyproject_save_files %{pypi_name}
+
+mkdir -p %{buildroot}%{_datadir}/%{pypi_name}/apache/
+install -m 644 apache/%{pypi_name}.wsgi %{buildroot}%{_datadir}/%{pypi_name}/apache/%{pypi_name}.wsgi
+install -m 644 apache/%{pypi_name}.conf %{buildroot}%{_datadir}/%{pypi_name}/apache/%{pypi_name}.conf
+
+mkdir -p %{buildroot}%{_sysconfdir}/%{pypi_name}/
+install -m 644 apache/%{pypi_name}.cfg %{buildroot}%{_sysconfdir}/%{pypi_name}/%{pypi_name}.cfg
+
+%check
+%pyproject_check_import
+%{pytest} -v
+
+%files -n %{pypi_name} -f %{pyproject_files}
+%doc README.md %{pypi_name}/docs
+%license COPYING
+%config(noreplace) %{_sysconfdir}/%{pypi_name}/
+%config(noreplace) %{_datadir}/%{pypi_name}/apache/%{pypi_name}.conf
```

Comment:
There can't be configuration in `%{_datadir}`. Since it's basically an example, I think in can go in `%doc`.

Reply:
I was trying to install it to %{_docdir}, but then it seems it can't be mixed with using %doc, so I removed the .conf from the install and added it straight to the %doc section. Mock seems to be fine with it. Is it ok?

Reply:
Yes I think that's better, thanks.